TestProjects/Playground/Playground.Forms.iOS/Playground.Forms.iOS.csproj

```diff
@@ -114,11 +114,9 @@
   <ItemGroup>
     <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Contents.json">
       <Visible>false</Visible>
-      <InProject>false</InProject>
```

Comment:
Please revert all changes to iOS projects in this PR

Reply:
Waiting for answer at top comment.